.github/workflows/ci.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Install pygac
         shell: bash -l {0}
         run: |
-          pip install --no-deps -e .
+          pip install --no-deps -e .[georefencer]
```

Comment:
In pyproject.toml it's called `georef`, could this be a problem?

Reply:
Thanks for pointing this out, I had missed it!